tuf/ngclient/updater.py

```diff
@@ -185,8 +185,16 @@ def download_target(
         else:
             target_base_url = _ensure_trailing_slash(target_base_url)
 
-        target_filepath = targetinfo["filepath"]
         target_fileinfo: "TargetFile" = targetinfo["fileinfo"]
+        target_filepath = targetinfo["filepath"]
+        consistent_snapshot = False
+        if hasattr(self._trusted_set.root.signed, "consistent_snapshot"):
```

Comment:
Why is this check needed? "Root" initializes the attribute `consistent_snapshot` in its constrictor?

Reply:
Yes, you are right this check is incorrect.
I forgot that we are always setting consistent_snapshot even if it's nonexistent in the JSON file itself.